__main__.py

```diff
@@ -1,6 +1,8 @@
 import streamlit as st
 import pandas as pd
 import numpy as np
+import os
+from Phraends_Flask.static.read_text_file import read_text_file
```

Comment:
Since you will change the layout for `read_text_file.py`, you will also need to change the import line.